bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/internal/i18n/ResourceBundleTracker.java

```diff
@@ -30,16 +31,15 @@
  * @author Markus Rathgeb - Add locale provider support
  * @author Ana Dimova - fragments support
  */
-@SuppressWarnings("deprecation")
-public class ResourceBundleTracker extends ResolvedBundleTracker {
+@SuppressWarnings({ "deprecation", "rawtypes", "unchecked" })
```

Comment:
Yes, it really feels wrong to me to create a separate class just for this specific reason 😄